test/e2e/README.md

```diff
@@ -0,0 +1,6 @@
+Run the test from package root:
```

Comment:
When following this README, I get such errors. Should I do some additional steps?

<img width="1920" alt="Screenshot 2024-09-26 at 13 19 48" src="https://github.com/user-attachments/assets/f4c11c48-831b-4633-a914-cd2ce9b1207a">